engine/core/src/main/java/org/datacleaner/util/http/CASMonitorHttpClient.java

```diff
@@ -65,6 +65,10 @@
 
     private static final Logger logger = LoggerFactory.getLogger(CASMonitorHttpClient.class);
 
+    private static final String CDI_TICKET_HEADER = "CDI-ticket";
+    private static final String CDI_SERVICE_URL_HEADER = "CDI-serviceUrl";
+    private static final String CDI_USERID = "CDI-userId";
```

Comment:
You're right. Unfortunately that means that making something generically CAS-compliant is not feasible at the moment. CASMonitorHttpClient should become DataHubMonitorHttpClient.